src/common.jl

```diff
@@ -225,6 +225,15 @@ function _pdf(d::Distribution{ArrayLikeVariate{N}}, x::AbstractArray{<:Real,N})
     return exp(@inbounds logpdf(d, x))
 end
 
+function pdf(
+    d::Union{Distribution{ArrayLikeVariate{N}},Missing}, x::Any
```

Comment:
I think we should not add a method with such a signature. I think we should not allow `d::Missing`, and `x::Any` seems too arbitrary.